docs/plugins/babel7.md

```diff
@@ -0,0 +1,59 @@
+---
+id: babel7-plugin
```

Comment:
Please add this plugin to the list here too.
https://github.com/fuse-box/fuse-box/blob/c8958a7b23178e71b1057cca0555cb70cbe0b67c/website/sidebars.json#L39